pkg/kubelet/kubelet.go

```diff
@@ -795,6 +802,9 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 			Name:   kl.nodeName,
 			Labels: map[string]string{"kubernetes.io/hostname": kl.hostname},
 		},
+		Spec: api.NodeSpec{
```

Comment:
@roberthbailey Also I don't understand your earlier comment on Kubelet not watching pods? Kubelet on master node has to start those static pods, ie, master components, and create corresponding mirror pods, and saved them to etcd through kube-apiserver, right?